features/tracker-allowlist.json

```
@@ -468,6 +468,17 @@
                     }
                 ]
             },
+            "cmp.quantcast.com": {
+                "rules": [
+                    {
+                        "rule": "cmp.quantcast.com/choice/JYWDqeLS64fbt/www.express.co.uk/choice.js",
```

```suggestion
                        "rule": "cmp.quantcast.com",
```

Looks like there's other requests required. Fine to just exempt the hostname here.